apps/web/components/SearchInput/SearchInput.tsx

```diff
@@ -426,30 +452,35 @@ const Results = ({
                 News[] &
                 SubArticle[])
                 .slice(0, 5)
-                .map((item) => {
+                .map((item, i) => {
                   const { onClick, ...itemProps } = getItemProps({
-                    item: '',
+                    item:
```

Comment:
Would it make sense to create a an object for this, rather than doing string interpolation?

eg. 

```typescript
interface SidebarNavAction<T extends ('directNav' | 'submit')> {
  type: T
  href: string
}

---
item: {
  type: 'directNav',
  href: .. etc
}

// and then you could just have something like
const handleResultAction = (item: SiebarNavAction) => {
  if (item.type === 'directNav') {
    // etc
  } else {
    // something submit
  }
}
```

At least then the logic could be a little bit more centralized and you wouldn't have to be comparing and composing strings

Reply:
I actually tried that but downshift turns whatever `item` is into a string so it ends up being [object Object]... So I ended up going this sort of hacky way.

Reply:
Now I see why it ends up being a string 🤦 
I'm gonna unhackify this then 😁